docs/contributing/getting-started-developers.md

```diff
@@ -115,34 +133,38 @@ code --install-extension [extensionName].vsix
 
-After installation, use `ctrl+shift+p (Windows)` or `cmd+shift+p (Mac)` to open the Extension Launcher and select `Web Template Studio: Launch` to run the extension.
+After installation, use `Ctrl + Shift ⇧ + P` (Windows/Linux) or `Command ⌘ + Shift ⇧ + P` (Mac) in Visual Studio Code to open the Extension Launcher and select `Web Template Studio: Launch` to run the extension.
 
 ## Tests
 
 ### Client Tests
-Currently, we use Jest and React Testing Library for our unit tests which was set up by create-react-app.
+Currently, we use *Jest and React Testing Library* for our unit tests which was set up by `create-react-app`.
 
-To run the client tests, go to src/client and run `yarn test` to run all the tests. Running this execute tests in files that end with *.{spec,test}.{js,jsx,ts,tsx}.
+To run the client tests, go to `src/client` and run `yarn test` to run all the tests. Running this execute tests in files that end with *.{spec,test}.{js,jsx,ts,tsx}.
 
-More info:
-- Jest: https://jestjs.io/
-- React Testing Library: https://testing-library.com/docs/react-testing-library/intro
+#### More info:
+- [Jest](https://jestjs.io/)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro)
 
 ### Template Tests
-Currently we use scripts to test the generated code. The scripts are located in the folder ..\src\extension\src\scripts.
-Ther genrate-tests script generates all the different permutations of a fullstack project (without azure services).
-Afterwards the run-tests script installes the dependencies for the generated project, runs yarn start, yarn lint and yarn test on all the generated projects.
+Currently we use scripts to test the generated code. The scripts are located in [src/extension/src/scripts](../../src/extension/src/scripts).
+The **generate-tests** script generates all the different permutations of a Fullstack project (without azure services).
+Afterwards the **run-tests** script *installs the dependencies* for the generated project, runs *yarn start*, *yarn lint* and *yarn test* on all the generated projects.
 
-To run the template tests go to src/extension and run `yarn template-test` to run the tests.
+To run the template tests go to the [`extension` folder](../../src/extension) and run `yarn template-tests` to run the tests.
 
 
 ## Under the Hood
 
 The following notes are inspired by the [vscode-webview-react](https://github.com/rebornix/vscode-webview-react)
 repository by [rebornix](https://github.com/rebornix):
 
-- We inline `index.html` content in `src/extension/src/extension.ts` when creating the webview
+- We inline `index.html` content in `ext-src/extension.ts`  when creating the webview
```

Comment:
Not sure if this path is correct

Reply:
Probably we can remove this "Under the Hood" section entirely. See what you think about it... I don´t see much value on it... If you check [here](https://github.com/rebornix/vscode-webview-react#under-the-hood) . It just seems to be this text copied, but doesn´t seem to apply much value on our documentation